include/cuco/detail/static_map/static_map.inl

```diff
@@ -237,13 +237,21 @@ template <class Key,
           class ProbingScheme,
           class Allocator,
           class Storage>
-template <typename OutputIt>
-OutputIt
+template <typename KeyOut, typename ValueOut>
+std::pair<KeyOut, ValueOut>
 static_map<Key, T, Extent, Scope, KeyEqual, ProbingScheme, Allocator, Storage>::retrieve_all(
-  OutputIt output_begin, cuda_stream_ref stream) const
+  KeyOut keys_out, ValueOut values_out, cuda_stream_ref stream) const
 {
-  auto const is_filled = static_map_ns::detail::slot_is_filled(this->empty_key_sentinel());
-  return impl_->retrieve_all(output_begin, is_filled, stream);
+  // XXX: thrust counting iterator doesn't work if size_type is larger than 32-bit
```

Comment:
WTF, really??

Reply:
Seems like a false alarm. I'm removing the comment for now and see if anyone complains.